recipes-graphics/spark/pxcore-libnode_10.15.3+git.bb

```diff
@@ -0,0 +1,11 @@
+include pxcore-libnode.inc
+
+SRC_URI += " \
```

Comment:
Does that mean pxcode-libnode doesn't need to download sources?

Reply:
it including pxcore.inc, that added SRC repo path